README.md

```diff
@@ -52,55 +55,107 @@ accumulated remain preserved and accessible over time.
 4. Compile the project: `javac Main.java`
 5. Run the project: `java Main`
 
-## Abbreviations
+### Screens
 
-| Name              | Abbreviation |
-|-------------------|--------------|
-| Controller        | C            |
-| UseCaseInteractor | UCI          |
-| InputBoundary     | IB           |
-| InputData         | ID           |
-| OutputBoundary    | OB           |
-| OutputData        | OD           |
-| Presenter         | P            |
-| ViewBoundary      | VB           |
-| View              | V            |
-| Gateway           | G            |
+![session_load.png](images/session_load.png)
 
+![main_menu.png](images/main_menu.png)
+
+![month_menu.png](images/month_menu.png)
+
+![add_category.png](images/add_category.png)
+![edit_category.png](images/edit_category.png)
+
+![add_expense.png](images/add_expense.png)
+![edit_expense.png](images/edit_expense.png)
+
+![graphical_summary.png](images/graphical_summary.png)
+
+### Testing
+This project uses JUnit as its testing framework. All the tests are contained inside the `src/test/java` directory.
+It contains tests for the entity classes, use case interactor classes, and the file storage gateway. It doesn't contain
+any tests for most of our `view` package, since the classes contained in it are view, presenter, and controller classes.
+View classes contain UI elements that can't really be tested in any way other than by being looked at, and 
+presenter/controller classes only pass data around without much logic to be tested.
+
+To run tests: 
+1. Open the project in IntelliJ.
+2. Open the 'Project' panel on the left.
+3. Right-click `src/main/test` directory and select `Run 'Tests in 'course-project-DoorKeepers.test''`
+
+To run any specific testing class, navigate to it in the 'Project' panel in IntelliJ, right-click it, and 
+select `Run 'ClassNameTest'`.
+
+Test coverage can be seen below:
+
+![test_coverage.png](images/test_coverage.png)
+
+The full report is contained inside the `reports/coverageReport` directory.
+
+### JavaDoc
+The project uses the JavaDoc framework to provide documentation for all of our methods, classes, and interfaces. 
+This documentation is written into the code, but we also provide generated documentation pages inside our 
+`reports/JavaDoc` directory.
 
 ## Clean Architecture Specifications
```

Comment:
Maybe rename this to Implementation and also move the Testing and Javadoc sections to here? I feel like these two sections are not really a part of Usages. 

The Implementation section below could be renamed to Implementation Design or Implementation Structure.

Reply:
Oh that's a good idea. I could make an Implementation section with Code Structure (formerly Implementation), Testing, and JavaDoc as its subsections

Reply:
Yep! Look great!